config/test.exs

```diff
@@ -0,0 +1,20 @@
+use Mix.Config
+
+# Configure your database
+config :auth_mvp, AuthMvp.Repo,
+  username: "postgres",
+  password: "postgres",
+  database: "auth_mvp_test",
+  hostname: "localhost",
+  pool: Ecto.Adapters.SQL.Sandbox
+
+# We don't run a server during test. If one is required,
+# you can enable the server option below.
+config :auth_mvp, AuthMvpWeb.Endpoint,
+  http: [port: 4002],
+  server: false
+
+# Print only warnings and errors during test
+config :logger, level: :warn
+config :auth_mvp, :elixir_auth_google, AuthMvp.Mock.ElixirAuthGoogle
+config :auth_mvp, :elixir_auth_github, AuthMvp.Mock.ElixirAuthGithub
```

Comment:
This configuration allow us to select the mock package for the test

Reply:
But _why_ do we want to Mock these pages? 
Is there an issue with this is described can it be linked to using a comment in the code? 
# 💭

Reply:
I think we could get the urls if we are setting environment variables in Travis for the oauth app, i.e this part of the code:
https://github.com/dwyl/auth-mvp/blob/1232eafeba21ff993bc7b4e9feae7b78d4e8f4d1/lib/auth_mvp_web/controllers/auth_url_controller.ex#L9-L10

However I don't think it's possible to get a token and profile from the google/github as it requires user action on the consent screen to exchange the codes with tokens